src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,62 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
 public class WorkWithFile {
 
+    private int saply = 0;
+    private int buy = 0;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        getRead(fromFileName);
+        getWrite(toFileName);
+    }
+
+    public void getRead(String fromFileName) {
+        try (BufferedReader bufferedReader = Files.newBufferedReader(Paths.get(fromFileName))) {
+            String line;
+            while ((line = bufferedReader.readLine()) != null) {
+                String[] words = line.split(" ");
```

Comment:
```suggestion
                String[] words = line.split(",");
```
у csv файлі кожен елемент розділений комою, тому по цьмоу можна сплітити і по індексах вже достукуватись до клітинок